src/macros/externs.sem

```diff
@@ -21,20 +21,20 @@ extern setExtern: [getTypeName:[]:bytes32, addSetItem:[address]:int256, contains
 extern stackExtern: [initialize:[address]:int256, isEmpty:[]:int256, peek:[]:int256, pop:[]:int256, push:[int256]:int256]
 extern branchExtern: [fork:[]:bool, getChildBranch:[int256]:address, getCurrentReportingWindow:[]:address, getForkEndTime:[]:uint256, getForkingMarket:[]:address, getNextReportingWindow:[]:address, getParentBranch:[]:address, getParentPayoutDistributionHash:[]:bytes32, getPreviousReportingWindow:[]:address, getReportingPeriodDurationInSeconds:[]:uint256, getReportingWindowByMarketEndTime:[uint256,bool]:address, getReportingWindowByTimestamp:[uint256]:address, getReportingWindowId:[uint256]:address, getReputationToken:[]:address, getTopics:[]:address, getTypeName:[]:bytes32, initialize:[address,bytes32]:bool, isContainerForMarket:[address]:bool, isContainerForRegistrationToken:[address]:bool, isContainerForReportingToken:[address]:bool, isContainerForReportingWindow:[address]:bool, isContainerForShareToken:[address]:bool, isParentOf:[address]:bool]
-extern disputeBondTokenExtern: [allowance:[address,address]:uint256, approve:[address,uint256]:bool, balanceOf:[address]:uint256, getBondHolder:[]:address, getBondRemainingToBePaidOut:[]:uint256, getMarket:[]:address, getDisputedPayoutDistributionHash:[]:bytes32, getReputationToken:[]:address, initialize:[address,address,uint256,bytes32]:bool, totalSupply:[]:uint256, transfer:[address,uint256]:bool, transferFrom:[address,address,uint256]:bool, withdraw:[]:bool]
-extern registrationTokenExtern: [allowance:[address,address]:int256, approve:[address,uint256]:int256, balanceOf:[address]:uint256, getBranch:[]:int256, getPeakSupply:[]:int256, getReportingWindow:[]:int256, getReputationToken:[]:int256, initialize:[address]:int256, redeem:[]:int256, register:[]:int256, totalSupply:[]:int256, transfer:[address,uint256]:int256, transferFrom:[address,address,uint256]:int256]
+extern disputeBondTokenExtern: [allowance:[address,address]:uint256, approve:[address,uint256]:bool, balanceOf:[address]:uint256, getBondHolder:[]:address, getBondRemainingToBePaidOut:[]:uint256, getMarket:[]:address, getDisputedPayoutDistributionHash:[]:bytes32, getReputationToken:[]:address, initialize:[address,address,uint256,bytes32]:bool, getTotalSupply:[]:uint256, transfer:[address,uint256]:bool, transferFrom:[address,address,uint256]:bool, withdraw:[]:bool]
+extern registrationTokenExtern: [allowance:[address,address]:int256, approve:[address,uint256]:int256, balanceOf:[address]:uint256, getBranch:[]:int256, getPeakSupply:[]:int256, getReportingWindow:[]:int256, getReputationToken:[]:int256, initialize:[address]:int256, redeem:[]:int256, register:[]:int256, getTotalSupply:[]:int256, transfer:[address,uint256]:int256, transferFrom:[address,address,uint256]:int256]
```

Comment:
Note: We no longer need to update this file anymore as we don't really have any serpent of significance left.  Once the last few contracts are migrated from Serpent to Solidity we'll be deleting this file.

Reply:
Yeah, I figured it wasn't necessary, but I did it to keep things consistent until the file gets deleted.